src/Views/inputView.js

```diff
@@ -0,0 +1,15 @@
+import { Console } from '@woowacourse/mission-utils'
+import { ASK } from '../Constants/MESSAGE.js'
+
+const InputView = {
+	async visitDate() {
+		const input = await Console.readLineAsync(ASK.VISIT_DAY)
+		return input
+	},
```

Comment:
```suggestion
	visitDate() {
		return Console.readLineAsync(ASK.VISIT_DAY)
	},
```
이런 식으로 축약해도 똑같이 동작하지 않을까요?!